pkg/openapi/openapi_generated.go

```diff
@@ -1518,6 +1525,13 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 								Ref:         ref("github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1.ServiceInstancePropertiesState"),
 							},
 						},
+						"provisioned": {
```

Comment:
This has not been updated with the json name change.

I wonder if this is supposed to be caught by the portion of the CI that verifies that the generated files are up-to-date.